lib/saturn-node.js

```diff
@@ -1,29 +1,42 @@
 import timers from 'node:timers/promises'
 import { execa } from 'execa'
-import { join } from 'node:path'
 import { arch, platform } from 'node:os'
 import { fetch } from 'undici'
 import * as Sentry from '@sentry/node'
-
-const archOverwritten = platform() === 'darwin' ? 'x64' : arch()
+import { installBinaryModule, getBinaryModuleExecutable } from './modules.js'
+
+export async function install () {
+  await installBinaryModule({
+    module: 'saturn-l2-node',
+    repo: 'filecoin-saturn/L2-node',
+    distTag: 'v0.5.0',
```

Comment:
I slightly prefer to extract the dist tag into a named constant defined at the top of this source code file to make it easier to find.